AnkiDroid/build.gradle

```
@@ -168,6 +168,9 @@ dependencies {
     testImplementation "org.robolectric:robolectric:4.3.1"
     testImplementation 'androidx.test:core:1.2.0'
     testImplementation 'androidx.test.ext:junit:1.1.1'
+    //apparently required for testing fragments :/
+    //https://issuetracker.google.com/issues/122321150
```

Fascinating bug thread(s) there. For me I had to read those in order to answer the question I had, so I'd like the comment to distill it and answer the question directly, also the real tracking issue is different

```suggestion
    // debugImplementation required vs testImplementation: https://issuetracker.google.com/issues/128612536
```

Finally, I actually use standard lint rule packs in other languages and it's not a big deal but I wonder about your 'comment slashes with no space after' style - it's visually jarring to me and linted out via my standard packages. Is this important to you? If not, I humbly submit I prefer '// then comment' :-)

---

The horrors of being a self-taught programmer. Just wait until you catch me using Allman-style braces 😉

Can we start up a wiki page or issue about these nits? Better to encode these into autolint settings, rather than add PR comments.

---

Yes we can (go for it hahaha) - honestly it should all be in codacy, driven by a checkstyle or pmd xml that Android Studio reads. I think there actually is a coding standards document even? In the codebase?

---

https://github.com/ankidroid/Anki-Android/blob/master/docs/code_conventions/AnkiDroid_code_conventions.html ... I missed that.

---

... we should probably have an Anki deck for this